Header header
uint8 y_step
uint16[] y_ws
uint8 int_compress
uint8[] intensity
